default/scripting/species/common/troops.macros

```diff
@@ -39,10 +39,19 @@ EffectsGroup      // gives human bonuses when AI Aggression set to Beginner
             accountinglabel = "DIFFICULTY"
             priority = [[DEFAULT_PRIORITY]]
             effects = SetMaxTroops value = max(6, Value * 2)
+
+        EffectsGroup // increase 1 per turn, up to max
+            scope = Source
+            activation = And [
+                Planet
+                (Source.LastTurnConquered < CurrentTurn)
```

Comment:
LastTurnAttackedByShip?

Reply:
added

Reply:
There is currently no effect to increase troop current meter for outposts and there should be one.

Same for supply, could do so for all relevant meters, I suspect stealth and stockpile are likely affected.
I can look into this late tonight if you would rather.

Reply:
Stealth does not need an effect to increase it turn by turn. It just has a single meter not a separate max/target meter.

Reply:
Added some additional effects for outpost troops and supply.